clients/cloud/kotlin/src/main/kotlin/io/confluent/examples/clients/cloud/model/DataRecord.kt

```diff
@@ -0,0 +1,11 @@
+package io.confluent.examples.clients.cloud.model
+
+import com.google.gson.Gson
+
+data class DataRecord(var count: Long) {
+  constructor() : this(0L)
```

Comment:
I think it will make sense to remove this secondary constructor and add default value to count in primary constructor

Reply:
That will work. Thanks

Reply:
fixed